e2e-test/tsps_e2e_test.py

```diff
@@ -0,0 +1,395 @@
+from workspace_helper import create_workspace, delete_workspace
+from app_helper import create_app, poll_for_app_url
+import argparse
+import requests
+import os
+import json
+import uuid
+import time
+import logging
+
+
+# Upload data to WDS using APIs
+def upload_wds_data_using_api(wds_url, workspace_id, tsv_file_name, record_name):
```

Comment:
nittiest of nits: in other functions the `_using_api` is implied and not included in the function name, i would remove here